frontend/src/components/common/export-recipients/ExportRecipients.tsx

```diff
@@ -104,7 +104,16 @@ const ExportRecipients = ({
         content = content.concat(emptyExplanation)
       }
 
-      const sentAtTime = new Date(sentAt)
+      let sentAtTime
+      if (!sentAt) {
+        console.error(
+          'sentAt is undefined. Using current Date() as a fallback.'
+        )
+        sentAtTime = new Date()
```

Comment:
This fallback doesn't seem necessary in practice, but I think it may be good as a form of defensive programming.